lib/membrane_flv_plugin/parser.ex

```diff
@@ -59,8 +59,11 @@ defmodule Membrane.FLV.Parser do
     type = resolve_type(type)
     {type, codec, codec_params, payload} = parse_payload(type, payload)
 
+    timestamp = parse_timestamp(timestamp, timestamp_extended)
+
     packet = %Packet{
-      timestamp: parse_timestamp(timestamp, timestamp_extended),
+      pts: timestamp,
+      dts: timestamp,
```

Comment:
Why are they the same timestamps?  Also, I think that dts for audio type should be nil especially that Membrne.Buffer allows for this https://hexdocs.pm/membrane_core/0.8.1/Membrane.Buffer.html#get_dts_or_pts/1

Reply:
Yeah, but FLV timestamp is dts

Reply:
DTS cannot be nil here, because it maps to TAG timestamp. So if the struct allowed it to be skipped, that would mean that that TAG has no timestamp. As for PTS, it will soon be calculated from dts using composition_time (if present)